documentation/docs/getting_started/installation/docker_compose_setup.md

```diff
@@ -0,0 +1,122 @@
+
+```bash
+docker-compose --env-file .env up --build
+```
+It is totally normal to see `error : No root identity found!` while the containers are starting up. After startup there should be no remaining errors.
+Now you are done with the docker-compose setup. To make sure that everything works as expected read the next section.
+
+## Test your API
+
+By following this section you can check if the installation was successful.
+
+### Check Container
+
+First we want to test if all docker container started up correctly.
+
+List all container:
+
+```bash
+docker-compose ps
+```
+
+You should be able to see all container except `api_initialize-audit-trail_1` and `api_initialize-ssi-bridge_1` in a `Up` state.
+
+```
+                       Name                                     Command               State                       Ports                    
+-------------------------------------------------------------------------------------------------------------------------------------------
+integration-services_audit-trail-gw_1                       docker-entrypoint.sh node  ...   Up       3000/tcp                                    
+integration-services_initialize-audit-trail_1               docker-entrypoint.sh node  ...   Exit 0                                               
+integration-services_initialize-ssi-bridge_1                docker-entrypoint.sh node  ...   Exit 0                                               
+integration-services_ssi-bridge_1                           docker-entrypoint.sh node  ...   Up       3000/tcp                                    
+is-dashboard                                                docker-entrypoint.sh npm r ...   Up       0.0.0.0:3055->3000/tcp                      
+mongo                                                       docker-entrypoint.sh mongod      Up       0.0.0.0:27017->27017/tcp                    
+traefik                                                     /entrypoint.sh --api.insec ...   Up       0.0.0.0:3000->80/tcp, 0.0.0.0:8080->8080/tcp
+```
+
+### Check Endpoints
+
+Now you can check if you can reach the `audit-trail-gw` and `ssi-brdige` via http by running the following commands:
+
+```bash
+curl http://localhost:3000/audit-trail-gw/info
+```
+
+```bash
+curl http://localhost:3000/ssi-bridge/info
+```
+
+:::info
+The Swagger API documentation **cannot** be reached currently. This issue will be fixed in a future release.
+:::
+
+## Dashboard
+
+We also supply a dashboard with the Integration Services. You can reach the dashboard in your browser via:
+
+```bash
+http://localhost:3000
+```
+
+:::info
+To make requests from the dashboard to the api you have to have CORS enabled. For local testing purposes this can be done with a CORS browser plugin. On first load the dashboard might show a Javascript module error. This bug is not related to our implementation and can be fixed with a simple page reload.
```

Comment:
If we run via docker we don't need to enable cors this just applies when starting it via npm run dev.

Reply:
so imo I would remove this statement here

Reply:
we could add this to the specific dashboard section where we can show how to run via npm run dev